folsom/src/main/java/com/spotify/folsom/Metrics.java

```diff
@@ -38,6 +38,8 @@
    */
   void registerOutstandingRequestsGauge(OutstandingRequestsGauge gauge);
 
+  void unregisterOutstandingRequestsGauge(OutstandingRequestsGauge gauge);
```

Comment:
This technically breaks the API, but in practice only ones who implement the interface are shipped in packages that are versioned with the same number so probably ok anyway.

Reply:
Yeah, this seems okay without bumping major to me.